plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,6 +1,6 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-0.14.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.2.5.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-0.15.0.gem"
+GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.3.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/parse-cron-0.1.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/sinatra-2.0.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/rack-protection-2.0.4.gem"
```

Comment:
Could you update sinatra and rack-protecion here?

Reply:
Updated